app/src/main/java/com/doctor/yumyum/data/remote/RetrofitClient.kt

```diff
@@ -0,0 +1,32 @@
+package com.doctor.yumyum.data.remote
+
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object RetrofitClient {
+    fun getClient(): Retrofit {
+        val baseUrl = ""
+
+        val baseInterceptor: Interceptor = (Interceptor { chain ->
+            val builder = chain.request().newBuilder()
+                .build()
+            chain.proceed(builder)
+        })
+
+        val client = OkHttpClient.Builder()
+            .connectTimeout(100, TimeUnit.SECONDS)
+            .readTimeout(100, TimeUnit.SECONDS)
```

Comment:
```suggestion
            .connectTimeout(10, TimeUnit.SECONDS)
            .readTimeout(10, TimeUnit.SECONDS)
```
1. 타임아웃 시간으로 100초는 너무 긴듯 싶습니다. 적당한 시간대를 추천드립니다 (10초는 예시)
2. 상수로 빼내는 것을 권유드립니다 ㅎㅎ

Reply:
`object`파일 안에선 상수를 바로 선언해도 괜찮은가요?!?